cmd/kubectl-gadget/filetop.go

```diff
@@ -101,7 +101,7 @@ var filetopCmd = &cobra.Command{
 
 func init() {
 	filetopCmd.Flags().IntVarP(&maxRows, "maxrows", "r", 20, "Maximum rows to print")
-	filetopCmd.Flags().StringVarP(&sortByStr, "sort", "", "rbytes", "Sort column")
+	filetopCmd.Flags().StringVarP(&sortByStr, "sort", "", "rbytes", fmt.Sprintf("Sort column, possibles values are: %s", strings.Join(types.SortBySlice, ", ")))
```

Comment:
IMO "possibles values are" is too verbose, the following is a bit shorter and understandable as well. 

```suggestion
	filetopCmd.Flags().StringVarP(&sortByStr, "sort", "", "rbytes", fmt.Sprintf("Sort column: (%s)", strings.Join(types.SortBySlice, ", ")))
```

Reply:
This is how it's printed in `docs/gadgets/filetop.md` too, I think those two output should be aligned as much as possible.

Reply:
Thinking to it, it is also the way output possibles values are printed:

```
-o, --output string          Output format (columns, json, custom-columns). (default "columns")
```

So let's stick to this one, it we want to change it, we would do it later.